D/readme.txt

```diff
@@ -57,7 +57,9 @@ use.
 
 If you wish to make use of the emulated Star's Ethernet interface, you
 will need to have WinPCAP installed (on Windows) or libpcap (on the Unix of 
-your choice).  WinPCAP can be downloaded from http://www.winpcap.org/.
+your choice).  WinPCAP can be downloaded from http://www.winpcap.org/.    
+If only Dodo emulated XNS services are to be accessed from Darkstar, a direct
+connection to the Dodo NetHub can be configured instead of using a real network. 
```

Comment:
Suggest adding a section somewhere covering Dodo (what it is, where to get it, etc...)